app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,8 @@
 
         <activity
             android:name=".PlayerActivity"
-            android:screenOrientation="userLandscape" />
+            android:configChanges="screenSize|smallestScreenSize|screenLayout|orientation|keyboard|keyboardHidden|uiMode"
```

Comment:
I don't know exactly what this does. I read that if you set values here you have to manually handle configuration changes in the activity. 

Could you explain what exactly this does and why?

Reply:
I forgot to remove that line. Yes this for manually handle configuration which is not needed.